packages/custom-token-bridging/scripts/exec-viaDapp.js

```diff
@@ -136,47 +128,14 @@ const main = async () => {
 
   const registerTokenRec = await registerTokenTx.wait()
   console.log(
-    `Registering token txn confirmed on L1! 🙌 ${registerTokenRec.transactionHash}`
+    `Registering token txn confirmed on L1! 🙌 ${registerTokenRec.transactionHash}`,
+    'You can now go ahead and transfer(deposit) your custom token to L2'
```

Comment:
We should clarify here that only the L1 side is confirmed (i.e., the retryable hasn't yet been redeemed); i.e.,  they can't deposit yet